java/src/main/java/com/google/appengine/tools/mapreduce/outputs/BigQueryGoogleCloudStorageStoreOutput.java

```diff
@@ -25,20 +26,21 @@
   private final MarshallingOutput<O, GoogleCloudStorageFileSet> dataMarshallingOutput;
   private final BigQueryMarshaller<O> bigQueryMarshaller;
   private final String bucketName;
-  private final String fileNamePattern;
+  private final String jobName;
```

Comment:
The change is good, however do you mind to remove jobName and bucketName (they are both private and seems that they are not using outside of the constructor)....